app/models/profit_share_pass.rb

```
@@ -169,4 +169,36 @@ def make_scenario(
       )
     end
   end
+
+  def efficiency_cap_from_snapshot
+    (snapshot && snapshot["inputs"]["efficiency_cap"]) || nil
+  end
+
+  def desired_buffer_months
+    (snapshot && snapshot["inputs"]["desired_buffer_months"]) || nil
+  end
+
+  def gross_revenue 
+    (snapshot && snapshot["inputs"]["actuals"]["gross_revenue"]) || nil
+  end
+
+  def gross_expenses
+    (snapshot && snapshot["inputs"]["actuals"]["gross_expenses"]) || nil
+  end
+
+  def gross_payroll
+    (snapshot && snapshot["inputs"]["actuals"]["gross_payroll"]) || nil
+  end
+
+  def projected_monthly_cost_of_doing_business
+    (snapshot && snapshot["inputs"]["projected_monthly_cost_of_doing_business"]) || nil
+  end
+
+  def fica_tax_rate
+    (snapshot && snapshot["inputs"]["fica_tax_rate"]) || nil
+  end
+
+  def internals_budget_multiplier
+    (snapshot && snapshot["inputs"]["internals_budget_multiplier"]) || nil
+  end 
```

A few notes here:
- There's inconsistency in the naming convention of these methods, some end with `_from_snapshot` and some don't
- we shouldn't fallback to `nil`, rather a type that corresponds to the type returned in a good case (so the FE doesn't need to do nil checks (this is more of a sanctuary best practice that ruby dogma)
- a good way to dig through hashes with string keys in Ruby is using `dig`, for example: `snapshot.dig("inputs", "fica_tax_rate") || 0`
- a good way to wrap a chained method in a nil check is to use `try`, for example: `snapshot.try(:dig, ["inputs", "fica_tax_rate"]) || 0`
- that said, Ruby's unsafe access operator probably beats `try` because wrapping a chained method feels unergonomic, for example: `snapshot&.dig("inputs", "fica_tax_rate") || 0`
- finally, logic used **soley** for rendering a model should be in the serializer file, not the model file (your serializer methods can do the above digging directly, rather than calling a model method)!

---

Sorry @hhff - this code wasn't ready for re-review, I was like "just make it work" mode after hooking it up to the ember side. Your notes are v helpful nonetheless and I'll be sure to address them! 

Marking this PR as draft until it's actually ready for your eyes again. Thanks!

---

Re fallback vals: `to_f` takes care of this for us (which we need to do anyway bc some snapshots start storing values as strings instead of floats). Not as explicit as `|| 0` but it works and I think `.to_f` _and_ a fallback would be misleading about `to_f` behavior. I can add a comment if this isn't obvious to Ruby devs?

```
irb(main):001:0>  nil.to_f
=> 0.0
```